src/DataTable/DataTable.scss

```diff
@@ -22,6 +22,10 @@ $data-table-pagination-dropdown-min-width: 6rem !default;
     border-bottom-left-radius: $border-radius;
     border-bottom-right-radius: $border-radius;
   }
+
+  * {
+    font-size: $font-size-sm;
```

Comment:
@monteri Looking over the Figma spec for `DataTable`, it seems we may want to be a bit more selective in what we set as `$font-size-sm`.

Not _all_ the content within DataTable is set as small, but its more focused around the content of the table that should have `$font-size-sm`, not everything. For example, the font sizes within the filters and actions stay normal size, but the "Showing 1 - 50..." both above and below the table contents and the table contents themselves should have `$font-size-sm`.

I'd probably just crosscheck with the [Figma spec](https://www.figma.com/file/eGmDp94FlqEr4iSqy1Uc1K/Paragon-Design-System?node-id=10469%3A37756) on which parts of `DataTable` should be small (i.e., by clicking on an element in Figma, you can see what should be small in the sidebar, see screenshot).

![image](https://user-images.githubusercontent.com/2828721/148392122-797be9c9-f839-4bc3-8f20-3c002bfe2be5.png)

Should probably have been more specific in the acceptance criteria ;)

Reply:
Yes, thanks for clarification, I thought the ticket was meant to make whole text with the small text size :)
Now I placed `font-size: $font-size-sm;` only for the wrapper so it made the expected result as in the Figma

Reply:
Awesome, thanks for the update!